application/src/main/java/net/corda/v5/application/interop/facade/FacadeMethodType.java

```diff
@@ -0,0 +1,6 @@
+package net.corda.v5.application.interop.facade;
+
+public enum FacadeMethodType {
```

Comment:
I'd remove this file and add the enum to FacadeMethod.java - cannot see a reason why users of the enum would not import FacadeMethod anyway